### PR TITLE
pyvmi can build and example can work

### PR DESCRIPTION
**Pull request by Badangel**
I modify the pyvmi.c. 
1.the pyvmi_init_complete can work now, The example (process-list.py) not use function init but use function init_complete
2.Notes some error funtion make the pyvmi.py can temporary build and install。
notes funtion:
`get_access_mode
get_page_mode
v2pcache_flush`

### CODE REVIEW

**Comment by Badangel** [comment]
I test the example can work.

File: tools/pyvmi/pyvmi.c [hunk 1]
```diff
@@ -135,48 +151,62 @@ pyvmi_init(
...
-        flags |= VMI_AUTO | VMI_INIT_COMPLETE;
-    }
-    else if (strcmp("partial", inittype) == 0) {
-        flags |= VMI_AUTO | VMI_INIT_PARTIAL;
-    }
-    else {
-        PyErr_SetString(PyExc_ValueError,
-                        "Inittype must be 'complete' or 'partial'");
+
+    // if (VMI_FAILURE == vmi_init_custom(&(vmi(object)), flags, vmiconfig)) {
```
Comment (tklengyel) [inline tools/pyvmi/pyvmi.c]: I think it's safe to just drop this completely now instead of commenting out.

**Comment by Badangel** [comment]
I drop those lines. But I still comment out the three funtion.
`get_access_mode get_page_mode v2pcache_flush`
Maybe someone can make three funtion work, I don't understand it deeply.
I wait for someone can make pyvmi build until I do it by myself.

**Comment by Wenzel** [comment]
Hi,
new python bindings based on CFFI have been merged upstream: #532 .
please take a look !

**Comment by smaresca** [comment]
closing in light of #532

**Pull request closed without merging**